Amigolden/src/app/pages/confirm-payment/confirm-payment.page.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-confirm-payment',
+  templateUrl: './confirm-payment.page.html',
+  styleUrls: ['./confirm-payment.page.scss'],
+})
+export class ConfirmPaymentPage implements OnInit {
+
+  constructor() { }
```

Comment:
This probably should just be a component instead of a page, that way you you don't need to affect the routing or page base.   Add an @Input Payments, so the html can run off the mocks